tempesta_fw/sock_clnt.c

```diff
@@ -212,9 +212,30 @@ tfw_sock_clnt_drop(struct sock *sk)
 	return 0;
 }
 
+/*
+ * The hook is executed when a client connection is closed by either
+ * side of the connection.
+ */
+static int
+tfw_sock_clnt_drop(struct sock *sk)
+{
+	return tfw_sock_clnt_do_drop(sk, "connection lost");
+}
+
+/*
+ * The hook is executed when a client connection is terminated due to
+ * an error of any kind.
+ */
+static int
+tfw_sock_clnt_error(struct sock *sk)
+{
+	return tfw_sock_clnt_do_drop(sk, "connection error");
+}
+
```

Comment:
Please don't introduce dummy wrappers which do exactly the same things except bit different debug output. Normal closing and error closing are treated the same way. That doesn't seem right, but this is how the things work now. We'll need to rethink this...